src/module.api/token.controller.ts

```diff
@@ -74,6 +74,6 @@ function mapTokenData (id: string, tokenInfo: TokenInfo): TokenData {
     minted: tokenInfo.minted.toFixed(),
     creation: { tx: tokenInfo.creationTx, height: tokenInfo.creationHeight.toNumber() },
     destruction: { tx: tokenInfo.destructionTx, height: tokenInfo.destructionHeight.toNumber() },
-    collateralAddress: tokenInfo.collateralAddress
+    ...(!(tokenInfo.collateralAddress === '') && { collateralAddress: tokenInfo.collateralAddress })
```

Comment:
```suggestion
    collateralAddress: tokenInfo.collateralAddress !== '' ? tokenInfo.collateralAddress : undefined
```

I think it's cleaner if you write it like this, instead of object restructuring and structuring.